react/introduction/intro-react/react-what-i-need-to-use-react.md

```diff
@@ -22,7 +22,7 @@ aspects:
 ---
 ## Content
 
-Being a truly flexible library, you can create applications from scratch with **React**, but also gradually including it to your codebase.
+Being a truly flexible library, you can not only create applications from scratch with **React**, but also gradually include them in to your codebase.
```

Comment:
Technically, you would gradually introduce React to the codebase, not applications.

Maybe say?

```suggestion
Being a truly flexible library, you can not only create applications from scratch with **React** but also gradually include it into your codebase.
```